pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorSubscriptionTest.java

```diff
@@ -704,6 +705,53 @@ public void testReplicatedSubscriptionWhenReplicatorProducerIsClosed() throws Ex
         Awaitility.await().untilAsserted(() -> assertNotNull(topic2.getSubscription(subscriptionName)));
     }
 
+    @Test
+    public void testReplicateSubBackLog() throws Exception {
```

Comment:
Please add another test:
- Enable replication
- Pub&Sub messages
- Disabled replication
- Ensure the Replicated Subscription Controller has been closed